### ISSUE TITLE
FlopCounterMode ignores custom operators

### ISSUE THREAD

**Issue by zou3519** [pytorch/pytorch#131041]
Today, FlopCounterMode silently ignores custom operators, instead pretending they don't contribute to the flop count. We should do at least one of the following:
- FlopCounterMode should warn on custom operators it doesn't know. Ideally it would do this only once to avoid spammy logs
- FlopCounterMode should error out on custom operators it doesn't know how to handle.

I don't have a good sense of which ones of the above we should do (I'm not sure how people use FlopCounterMode) and there are BC concerns. In general I would have started with the error because we are potentially lying to the user about their flop count when custom ops are involved. Thoughts?

cc @Chillee @ezyang @albanD

**Comment by zou3519** [comment]
Duplicate of https://github.com/pytorch/pytorch/issues/123800
